include/sys/abd.h

```diff
@@ -1,14 +1,23 @@
 /*
- * This file and its contents are supplied under the terms of the
- * Common Development and Distribution License ("CDDL"), version 1.0.
```

Comment:
Looks like they checked in with newer cddl 1.0 header, can you correct pls   https://github.com/openzfs/openzfs/blob/master/usr/src/uts/common/fs/zfs/sys/abd.h